MAINTAINERS.md

```diff
@@ -126,23 +221,19 @@ in this branch. The release tags will be human-meaningful, the release branch ne
   scripts/have-news HEAD^ > have-news.md
   ```
 
-- [ ] Create the release PR.
-
-  The above should have pushed state to GitHub to let you create a
-  new PR to merge the release branch back to the main branch. The PR
-  name should follow the convention of previous releases, which is
-  currently `chore(release): publish _release_label_`.  Paste
-  `have-news.md` as the description of the PR.  Follow the example
-  of previous releases for any other details of the PR.
+- [ ] Create the release PR:
```

Comment:
In my release PRs I have been trying to include hidden in a `details` tag, the formatted rebase todo of `master` since the last release was cut, so we can more easily see which PRs were included, and which weren't, and in the future, just reformat since that point only to avoid duplicate work.

Reply:
Can you elaborate?